### PR TITLE
libjpeg: add version 9d

### PR DESCRIPTION
**Pull request by mvanwaveren**

### CODE REVIEW

**Review by adamjstewart — COMMENTED** [review]
Can you sort the versions from newest to oldest?

**Comment by mvanwaveren** [comment]
@adamjstewart Thanks for your comment, I will change the sort.

I have one unsuccesfull check I do not understand. This is the one with

`codecov/project — 80.98% (-2.14%) compared to e49383c`

Can you let me know what I should do to pass this check ?

**Comment by adamjstewart** [comment]
For the coverage test, see the note at the bottom of https://spack.readthedocs.io/en/latest/contribution_guide.html#coverage.

**Comment by mvanwaveren** [comment]
@adamjstewart I updated the package. Can you see it?

I see that the tests are automatically launched, so my modification has been picked up.

File: var/spack/repos/builtin/packages/libjpeg/package.py [hunk 1]
```diff
@@ -12,12 +12,13 @@ class Libjpeg(AutotoolsPackage):
...
     homepage = "http://www.ijg.org"
-    url      = "http://www.ijg.org/files/jpegsrc.v9c.tar.gz"
+    url      = "http://www.ijg.org/files/jpegsrc.v9d.tar.gz"
 
+    version('9d', sha256='6c434a3be59f8f62425b2e3c077e785c9ce30ee5874ea1c270e843f273ba71ee')
     version('9c', sha256='650250979303a649e21f87b5ccd02672af1ea6954b911342ea491f351ceb7122')
     version('9b', sha256='240fd398da741669bf3c90366f58452ea59041cacc741a489b99f2f6a0bad052')
     version('9a', sha256='3a753ea48d917945dd54a2d97de388aa06ca2eb1066cbfdc6652036349fe05a7')
-
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/libjpeg/package.py]: Can you remove this space? It will cause the flake8 tests to fail

Comment (mvanwaveren) [reply var/spack/repos/builtin/packages/libjpeg/package.py]: Done.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**